src/test/java/emissary/core/FTestMovingAgent.java

```diff
@@ -129,7 +129,9 @@ private void runTest(@Nullable final String agentClass) {
             try {
                 Thread.sleep(250);
             } catch (Exception ex) {
-                // empty catch block
+                if (ex instanceof InterruptedException) {
+                    Thread.currentThread().interrupt();
+                }
```

Comment:
Any reason not to just change the caught exception type to `InterruptedException`?  (ie., `catch (InterruptedException ex) {`